front/pages/_app.tsx

```diff
@@ -34,6 +36,64 @@ if (DATADOG_CLIENT_TOKEN) {
   });
 }
 
+// Developer console recruitment message.
+if (
+  typeof window !== "undefined" &&
+  typeof console !== "undefined" &&
+  NODE_ENV === "production"
+) {
+  (() => {
+    try {
+      const alreadyShown = localStorage.getItem(CONSOLE_MESSAGE_SHOWN_KEY);
+      if (!alreadyShown) {
+        localStorage.setItem(CONSOLE_MESSAGE_SHOWN_KEY, "true");
+
+        console.log(
+          "%c" +
+            "██████╗ ██╗   ██╗███████╗████████╗\n" +
+            "██╔══██╗██║   ██║██╔════╝╚══██╔══╝\n" +
+            "██║  ██║██║   ██║███████╗   ██║   \n" +
+            "██║  ██║██║   ██║╚════██║   ██║   \n" +
+            "██████╔╝╚██████╔╝███████║   ██║   \n" +
+            "╚═════╝  ╚═════╝ ╚══════╝   ╚═╝   ",
+          "color: #54B47D; font-family: monospace; font-size: 12px; font-weight: bold;"
+        );
+
+        console.log(
+          "%c🚀 Hey there, curious developer!",
+          "color: #418B5C; font-size: 20px; font-weight: bold; margin: 10px 0;"
+        );
+
+        console.log(
+          "%cWe're creating a new AI operating system that has the potential to change how companies operate.\n\n" +
+            "Our mission at Dust is to transform how work gets done by letting any team\n" +
+            "and employee shape the exact agents they need to accelerate their jobs.\n\n" +
+            "Want to help us build this future? We're looking for talented engineers who:\n" +
+            "  • Are passionate about crafting exceptional developer experiences\n" +
```

Comment:
Are passionate about crafting rock-solid code and exceptional experiences at warp speed.